### DISCUSSION TITLE
Ignore vite requests

### DISCUSSION THREAD

**Discussion by unlocomqx** [cypress-io/cypress#28666] [Questions and Help]
Hi
The side panel is flooded with requests from vite

![CleanShot 2024-01-09 at 12 45 14](https://github.com/cypress-io/cypress/assets/793712/e38b5bc6-4ce5-4781-9715-ccbcccc9fc7e)

Is there a way to ignore these requests? 

Thank you

**Comment by portyu9** [comment]
Hi @unlocomqx 

On current Cypress, the easiest solution is the **Hide HTTP Requests** option in the Command Log menu. That was added in Cypress 15.4.0 and is useful when development-server traffic such as Vite requests is making the log noisy.

If you only want to suppress particular requests rather than all HTTP requests, `cy.intercept()` also supports disabling logging for matched traffic:

```js
cy.intercept('**/@vite/**', { log: false })
```

You can add additional patterns for the specific Vite URLs present in your application.

So:

- **Hide all HTTP request noise:** use the Command Log's **Hide HTTP Requests** setting.
- **Hide only selected Vite requests:** use `cy.intercept(..., { log: false })`.

Please mark this answer as accepted if it helped, thank you.
